app/models/screening.rb

```diff
@@ -3,4 +3,5 @@
 class Screening < ApplicationRecord
   belongs_to :movie
   belongs_to :cinema_hall
+  has_many :tickets
```

Comment:
You can define https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent here :wink: